buildpipeline/Core-Setup-PortableLinux-x64-BT.json

```diff
@@ -0,0 +1,942 @@
+    },
+    "PB_DebianKeys": {
+      "value": "/p:DebianId_ubuntu1404-x64=$(PB_DebianId_ubuntu1404-x64) /p:DebianId_debian8-x64=$(PB_DebianId_debian8-x64) /p:DebianId_ubuntu1604-x64=$(PB_DebianId_ubuntu1604-x64) /p:DebianId_ubuntu1610-x64=$(PB_DebianId_ubuntu1610-x64)"
+    },
+    "DockerTag_Ubuntu1404": {
+      "value": "ubuntu-14.04-debpkg-e5cf912-20175003025046"
+    },
+    "DistroRid_Ubuntu1404": {
+      "value": "ubuntu.14.04-x64"
+    },
+    "DockerImageName_Ubuntu1404": {
+      "value": "$(PB_DockerRepository):$(DockerTag_Ubuntu1404)"
+    },
+    "DockerCommonRunArgs_Ubuntu1404": {
+      "value": "--name $(PB_DockerContainerName)$(DockerTag_Ubuntu1404) -v \"$(PB_SourcesDirectory):$(PB_GitDirectory)\" -v $(Build.StagingDirectory)/sharedFrameworkPublish/:/root/sharedFrameworkPublish/ -w=\"$(PB_GitDirectory)\" $(DockerImageName_Ubuntu1404)"
+    },
+    "DockerTag_Ubuntu1604": {
+      "value": "ubuntu-16.04-debpkg-e5cf912-20174703024721"
+    },
+    "DistroRid_Ubuntu1604": {
+      "value": "ubuntu.16.04-x64"
+    },
+    "DockerImageName_Ubuntu1604": {
+      "value": "$(PB_DockerRepository):$(DockerTag_Ubuntu1604)"
+    },
+    "DockerCommonRunArgs_Ubuntu1604": {
+      "value": "--name $(PB_DockerContainerName)$(DockerTag_Ubuntu1604) -v \"$(PB_SourcesDirectory):$(PB_GitDirectory)\" -v $(Build.StagingDirectory)/sharedFrameworkPublish/:/root/sharedFrameworkPublish/ -w=\"$(PB_GitDirectory)\" $(DockerImageName_Ubuntu1604)"
+    },
+    "DockerTag_Ubuntu1610": {
+      "value": "ubuntu-16.10-debpkg-ec863bb-20170003030028"
+    },
+    "DistroRid_Ubuntu1610": {
+      "value": "ubuntu.16.10-x64"
+    },
+    "DockerImageName_Ubuntu1610": {
+      "value": "$(PB_DockerRepository):$(DockerTag_Ubuntu1610)"
+    },
+    "DockerCommonRunArgs_Ubuntu1610": {
+      "value": "--name $(PB_DockerContainerName)$(DockerTag_Ubuntu1610) -v \"$(PB_SourcesDirectory):$(PB_GitDirectory)\" -v $(Build.StagingDirectory)/sharedFrameworkPublish/:/root/sharedFrameworkPublish/ -w=\"$(PB_GitDirectory)\" $(DockerImageName_Ubuntu1610)"
+    },
+    "DockerTag_Debian8": {
+      "value": "debian-8.2-debpkg-9f87c3c-20173003023006"
+    },
+    "DistroRid_Debian8": {
+      "value": "debian.8-x64"
+    },
+    "DockerImageName_Debian8": {
+      "value": "$(PB_DockerRepository):$(DockerTag_Debian8)"
+    },
+    "DockerCommonRunArgs_Debian8": {
+      "value": "--name $(PB_DockerContainerName)$(DockerTag_Debian8) -v \"$(PB_SourcesDirectory):$(PB_GitDirectory)\" -v $(Build.StagingDirectory)/sharedFrameworkPublish/:/root/sharedFrameworkPublish/ -w=\"$(PB_GitDirectory)\" $(DockerImageName_Debian8)"
+    },
+    "CLI_NUGET_FEED_URL": {
+      "value": "https:%2F%2Fdotnet.myget.org/F/dotnet-core/api/v2/package"
+    },
+    "CLI_NUGET_SYMBOLS_FEED_URL": {
+      "value": "https:%2F%2Fdotnet.myget.org/F/dotnet-core/symbols/api/v2/package"
+    }
+  },
+  "demands": [
+    "Agent.OS -equals linux"
```

Comment:
Good catch. I agree: the build machine should have the oldest version of glibc that the portable build needs to support.